library/ws-widget/collection/src/lib/_common/content-toc/enroll-modal/enroll-modal.component.scss

```diff
@@ -1,40 +0,0 @@
-@import '_ws-vars.scss';
-
-.modal-container {
-    padding: 24px;
-
-    .radio-outer-div {
-        padding: 8px 16px;
-        border-radius: 4px;
-        border: 1px solid rgba(0, 0, 0, 0.16);
-        opacity: 1;
-    }
-
```

Comment:
This PR should not have any code related to TOC page, please confirm

Reply:
Same as above comment